app/components/view-epic-list/macro.test.js

```diff
@@ -0,0 +1,293 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+
+const macroWrapper = `{% from './components/view-epic-list/macro.njk' import viewEpicList %}
+                        {{ viewEpicList(params) }}`;
+
+describe('view-epic-list', () => {
+  describe('when type is met', () => {
+    it('should render the data when provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: [
+            {
+              id: 'C14E1',
+              name: 'access prescribable items',
+            },
+            {
+              id: 'C14E2',
+              name: 'manage Formularies',
+            },
+            {
+              id: 'C14E13',
+              name: 'access Patient Record',
+            },
+          ],
+          type: 'met',
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="some-data-identifier"] ul').length).toEqual(1);
+          expect($('[data-test-id="some-data-identifier"] li').length).toEqual(3);
+
+          done();
+        });
+    });
+
+    it('should not render the data when not provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: [],
+          type: 'met',
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="some-data-identifier"]').length).toEqual(0);
+
+          done();
+        });
+    });
+  });
+
+  describe('when type is not- met', () => {
```

Comment:
get rid of that little space after the `-`,
Also perhaps might be nice to put `not-met` in double quotes to indicate it means a thing. Same with the describe above put `met` in double quotes